### ISSUE TITLE
Error when exporting to onnx for albert-base-v2, issue with attention_mask

### ISSUE THREAD

**Issue by LearrningLukeLondon** [pytorch/pytorch#102631]
### 🐛 Describe the bug

When attempting to export the following pretrained onnx model below, which uses [albert-base-v2](https://huggingface.co/albert-base-v2), I get an error about not being able to unsqueeze the attention mask even though it is clearly a list AND the model does original model outputs as expected.

Code:
```
input_ids = [2, 184, 20, 745, 2667, 11318, 18, 37, 4335, 4914,3]
token_type_ids = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
attention_mask = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]

input_ids_tensor = torch.tensor([input_ids])
token_type_ids_tensor = torch.tensor([token_type_ids])
attention_mask_tensor = torch.tensor([attention_mask])

cur_state_dict = torch.load("./albert_model/checkpoint-69203/pytorch_model.bin", map_location="cpu")

model = AutoModelForSequenceClassification.from_pretrained("./albert_model/checkpoint-69203")

model.load_state_dict(cur_state_dict, strict=False)
model.eval()

with torch.no_grad():
    output = model(input_ids_tensor, token_type_ids_tensor, attention_mask_tensor)
    print("output: ", output)

    torch.onnx.export(model, 
                    (input_ids_tensor, token_type_ids, attention_mask), 
                    "new_onnx_model", 
                    export_params=True, 
                    opset_version=16,
                    do_constant_folding=True,
                    input_names = ["input_ids", "token_type_ids", "attention_mask"],
                    output_names = ["logits", "scores"])
```

The error message for the "torch.onnx.export" doesn't make sense because the "attention_mask" should be a list
<img width="598" alt="image" src="https://github.com/pytorch/pytorch/assets/135020113/54ad99f3-b711-490f-bad1-a9cdc4619195">

### Versions

Collecting environment information...
PyTorch version: 1.13.1+cu116
Is debug build: False
CUDA used to build PyTorch: 11.6
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.5 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.31

Python version: 3.8.10 (default, Mar 13 2023, 10:26:41)  [GCC 9.4.0] (64-bit runtime)
Python platform: Linux-5.0.0-1032-azure-x86_64-with-glibc2.29
Is CUDA available: True
CUDA runtime version: 11.1.105
CUDA_MODULE_LOADING set to: LAZY

Versions of relevant libraries:
[pip3] numpy==1.23.5
[pip3] pytorch-nlp==0.5.0
[pip3] pytorch-pretrained-bert==0.6.2
[pip3] torch==1.13.1+cu116
[pip3] torchaudio==0.13.1+cu116
[pip3] torchvision==0.14.1+cu116
[conda] Could not collect

**Comment by BowenBao** [comment]
Eyeballing it seems like two issues:
1. Input order should be `input_ids_tensor, attention_mask_tensor, token_type_ids_tensor`
2. Example inputs passed to `torch.onnx.export` should be tuple of tensors, i.e. `torch.onnx.export(model, (input_ids_tensor, attention_mask_tensor, token_type_ids_tensor), ...)`.

**Comment by thiagocrepaldi** [comment]
closing as there was no repsonse from op
